Formula/pldiff.rb

```diff
@@ -0,0 +1,16 @@
+class Pldiff < Formula
+  desc "Compares plist files line by line"
+  homepage "https://github.com/scottrigby/pldiff"
+  url "https://github.com/scottrigby/pldiff/archive/1.0.4.tar.gz"
+  sha256 "0093027aee4ceb049902a9fcea28923477c87c2f88813e2370fb4c12195b9ccd"
+
+  depends_on "colordiff" => :recommended
+
+  def install
+    system "/bin/sh", "install.sh", prefix
```

Comment:
@woodruffw ah ok, i was looking at other existing formulas for examples and this is how some others have done it. So just to be clear, are you saying `sh` is already discoverable so I can can just do this?:
```diff
+    system "sh", "install.sh", prefix
```

Also I saw the `prefix` var worked properly – I got this from some existing Formulas I looked through – but from the [cookbook]() it looked as if it's recommended instead to use the `#{prefix}` syntax. Is there a standard, or best practice for Homebrew Forumlas about this?